jmbitcoin/jmbitcoin/amount.py

```diff
@@ -60,4 +60,5 @@ def sat_to_str_p(sat):
 
 
 def fee_per_kb_to_str(feerate):
-    return str(feerate) + " sat/vkB (" + str(feerate / 1000) + " sat/vB)"
+    return (str(int(feerate)) + " sat/vkB (" +
+        '%.1f' % (int(feerate / 100) / 10) + " sat/vB)")
```

Comment:
As per your third test case below, this doesn't round to the nearest 1d.p. as users would naturally expect. See here for a couple of ways you can do that: https://stackoverflow.com/a/3400970

Reply:
It will always round down, that is correct way to represent feerate IMO. As "1.999 sat/vB" will be lower priority than "2.000 sat/vB". I think it's better to be pessimistic than optimistic here.

Reply:
I'd strongly favour rounding in the normal sense, only because rounding down to 1d.p. will just not be what people expect. It's not a huge issue though, I admit.

Reply:
I actually lost casting return result to `int` when changing `estimate_fee_per_kb()`. Now restored that behaviour, so this is a no issue for now, as result will alwas be round sat/vB value. Core also doesn't return more precise values, so this was only result of our randomization afterwards. So, basically we always rounded down anyway. Still I think it's good to keep this change if behaviour is changed in the future.

Reply:
OK.

Reply:
Yes, I see, we rounded down already.